react/src/redux/books/actions.js

```diff
@@ -0,0 +1,38 @@
+import {getBooksService, getBookService} from '../../services/books';
+
+export const ActionTypes = {
+  GET_BOOKS: 'GET_BOOKS',
+  GET_BOOK: 'GET_BOOK',
+  SET_BOOK_FILTER_TYPE: 'SET_BOOK_FILTER_TYPE',
+  SET_BOOK_FILTER_TEXT: 'SET_BOOK_FILTER_TEXT',
+};
+
+export const getBooks = () => {
```

Comment:
What we usually do is to declare an `actionCreators` object and export it as `default`. That object usually contains as fields these functions:
```
const actionCreators = {
  getBooks: () => { ... },
  getBook: id => { ... }
};

export default actionCreators;
```